### PR TITLE
Hotfix/homepage cleanup

### PR DESCRIPTION
**Pull request by shadowlion**
Yay! I wrote some typescript!

I did numerous small changes to `/` and `/about`, based on what I wrote for #136.

I felt like, when approaching the public domain, people don't really need to see the jargon of tech specs outlined from our repos; it's a system for anyone to use, including those non-technical. Therefore, we need to cater to a much wider audience with our public-interfacing pages.

Hopefully, this PR is just the beginning.

### CODE REVIEW

**Comment by yubing24** [comment]
@shadowlion Thank you for those changes. You probably want to check your tslint and fix the double quotes. Here is my tslint:
```
{
  "defaultSeverity": "error",
  "extends": ["tslint-config-standard"],
  "rules": {
    "curly": true,
    "import-spacing": true,
    "indent": [true, "spaces"],
    "prefer-const": true,
    "quotemark": [true, "single"],
    "semicolon": [true, "always"],
    "space-before-function-paren": false,
    "triple-equals": [true, "allow-null-check"]
  }
}

```

File: components/Footer.vue [hunk 1]
```diff
@@ -4,47 +4,54 @@
...
-        <img
-          src="https://img.shields.io/github/license/DancesportSoftware/das-frontend.svg?style=flat-square"
-        >
-          <p>&copy; {{ year }} — &nbsp; Dancesport Software LLC</p>
-          <p>API v.{{ build }}</p>
+          <!-- <img
+            src="https://img.shields.io/github/license/DancesportSoftware/das-frontend.svg?style=flat-square"
+          >-->
+          <p>&copy; {{ new Date().getFullYear() }} Dancesport Software, LLC</p>
+          <!-- <p>API v.{{ build }}</p> -->
```
Comment (yubing24) [inline components/Footer.vue]: @shadowlion  Please bring the API version back. It's important for me to check if the backend is update correctly.

File: components/Footer.vue [hunk 2]
```diff
@@ -4,47 +4,54 @@
...
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator';
+import { Component, Vue } from "vue-property-decorator";
 @Component({
-  name: 'FooterComponent'
+  name: "FooterComponent"
 })
 export default class FooterComponent extends Vue {
-  build: string =  '';
```
Comment (yubing24) [inline components/Footer.vue]: @shadowlion revert the changes here please

File: components/Footer.vue (hunk 1 above)
Comment (shadowlion) [reply components/Footer.vue]: I made it such that it only shows in development, but not in production.

File: components/Footer.vue (hunk 2 above)
Comment (shadowlion) [reply components/Footer.vue]: done

**Comment by shadowlion** [comment]
@yubing24 Haven't been able to check why the autoformatter that i have in vscode keeps changing to double quotes, so I've been editing it in sublime for now. check to see if the new changes work.

**Comment by yubing24** [comment]
@shadowlion if you also have prettier extension installed, or if you have any auto-format on-save configured, they might override tslint configuration.

**Comment by shadowlion** [comment]
@yubing24 yeah, I think that's what's happening - I'll wrangle with it.
